src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -363,31 +363,31 @@
   "item.minecolonies.golden_bread": "Golden Bread",
   "item.minecolonies.chorus_bread": "Chorus Fruit Bread",
 
-  "item.minecolonies.scroll_tp": "Ultrasafe colony teleport scroll",
-  "item.minecolonies.scroll_tp.tip": "Safely teleports you to the set colony",
+  "item.minecolonies.scroll_tp": "Colony Portal Scroll",
+  "item.minecolonies.scroll_tp.tip": "Teleports you to the set colony",
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
-  "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
-  "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
-  "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
+  "item.minecolonies.scroll.colony.none": "None, set one with sneak + use",
+  "item.minecolonies.scroll_area_tp": "Colony Group Portal Scroll",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",
+  "item.minecolonies.scroll_buff": "Regeneration Scroll",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
-  "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
-  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
```

Comment:
Ok, I think I get why you are removing the wololo, but it makes me sad to lose the reference. This was one of the more funny recent additions.

Reply:
Yeah, sadly it doesn't really fit as that's not what wololo actually did

Reply:
Perhaps find a way to work it into the regeneration scroll?

Reply:
It doesn't regenerate either, it changes enemies to allies. Maybe if we add extra things we can do to raiders (like turning them into citizens) we can work it into there?

Reply:
Fair, though it invokes the AoE priest which did both conversion and healing, so regeneration would be the closest reference we have today.